frontend/src/common/hooks/useCategoryFilter/common/selectUtils.ts

```diff
@@ -0,0 +1,296 @@
+import { Filters } from "react-table";
```

Comment:
The functionality in this file was moved *mostly* unchanged from `useCategoryFilter`. I have marked the changes below.